### ISSUE TITLE
Accessibility: server-list for Voice-over-Screenreader on Mac is not usable

### ISSUE THREAD

**Issue by Michael1972** [jamulussoftware/jamulus#3456]
currently, in jamulus 3.11 for mac, the server-list isn't accessibility for users of the voice-over-screnreader. In most situation keyboard-navigation fails or is so unresponsive, that navigating in the server-list is nearly impossible. Until 3.9, there was a special accessible build from @chigkim. But due to a qt-bug the server-list-accessibility in later versions obviously is broken. So, in order to navigate the server-list with the keyboard, accessibility should be reimplemented in 3.12

**Comment by ann0see** [comment]
Thanks for noticing this. Do you know which Qt bug that is?

**Comment by ann0see** [comment]
Ok, I enabled VoiceOver and yes, navigating the connect dialog table is difficult. Tabbing works - but it doesn't speak the correct server name. 
I think we might want to think about how to redesign the connect dialog. That would be a way bigger change though - and probably something for Jamulus 4.

**Comment by ann0see** [comment]
Ok. I found the issue: https://github.com/jamulussoftware/jamulus/issues/490
I'll tag the other one instead as r3.12.0.

Please comment there again.
